src/server/gameEngine/gameEngine.spec.ts

```diff
@@ -1021,6 +1021,48 @@ describe('Game Action', () => {
             expect(newBoardState.players[0].cemetery).toEqual([spellCard]);
         });
 
+        it('handles nonsensical cardIds', () => {
```

Comment:
catching a new potential server-side bug and introducing tests for it to guard against it